polaris/ocean/tests/single_column/ideal_age/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from polaris import TestCase
+from polaris.ocean.tests.single_column.forward import Forward
+from polaris.ocean.tests.single_column.initial_state import InitialState
+from polaris.ocean.tests.single_column.viz import Viz
+from polaris.validate import compare_variables
+
+
+class IdealAge(TestCase):
+    """
+    The default test case for the single column test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+    def __init__(self, test_group, resolution, ideal_age=True):
```

Comment:
```suggestion
    def __init__(self, test_group, resolution):
```
I don't think `ideal_age` is needed here as a parameter since it's always `True`.

Reply:
removed